gradle/dependencies.gradle

```diff
@@ -65,19 +65,18 @@ versions += [
   argparse4j: "0.7.0",
   avro: "1.8.1",
   avroPlugin: "0.10.0",
-  bcpkix: "1.61",
+  bcpkix: "1.62",
   checkstyle: "8.20",
   commonsCli: "1.4",
   gradle: "5.4.1",
   gradleVersionsPlugin: "0.21.0",
   grgit: "3.1.1",
-  httpcomponents: "4.5.7",
-  httpclient: "4.5.8",
-  httpmime: "4.5.2",
+  httpcomponents: "4.5.9",
```

Comment:
This `httpcomponents` is unused and not in AK.

Reply:
It's used here, but not in AK. I'll reduce the divergence.